assets/tests/TaskSchemaValidationServiceSpec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the TaskSchemaValidationService.
+ */
+
+describe('TaskSchemaValidationService', function() {
+  var QuestionDataService;
+  // Alias for TaskSchemaValidationService.
+  var Tsvs;
+  var questions = [];
+  // Should contain all question IDs.
+  // TODO(eyurko): Figure out a way to dynamically check to make sure
+  // that all question IDs are specified.
+  var QUESTION_IDS = ['reverseWords', 'parens', 'i18n', 'rle'];
+  globalData.questionSets.all = {};
```

Comment:
Sorry, I thought I did that as well. I must have forgotten to save before a commit.